submissions/rubyAdeniregun/todo-app/src/components/AddTodo.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react'; 
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faPlus } from "@fortawesome/free-solid-svg-icons";
+
+class AddTodo extends React.Component {
+
+  constructor() {
+    super();
+
+    this.state = {
+      todo: ""
+    };
+  }
+
+  render () {
+
+    const { todo } = this.state;
+    const enabled =
+          todo.length >=1          
+        
+    return (
+      <div className="addTodoContainer">
+        <h1>TODO</h1>
+        <form className="formbox" onSubmit={(e) => this.submitTodo(e)}>
+          <input id="addTodoInput" placeholder="create a new todo..." onChange={(e) => this.updateInput(e)} type="text"></input>
+          <button type="submit" className="btn" disabled={!enabled}>
+          <FontAwesomeIcon icon={faPlus} size='lg' className="icon"/>
```

Comment:
You don't need to install icons just because of `+` and `-` You can use the ones on your keyboard. Since this is only used once in code, it is unnecessary. This will impact the loading speed of your page. I think you should remove it @rubyade